rules/sun_checks.xml

```diff
@@ -243,7 +243,7 @@
       <property name="ignoreStatic" value="true"/>
       <property name="ignoreFinal" value="false"/>
       <property name="allowedAbbreviationLength" value="0"/>
-      <property name="allowedAbbreviations" value="IT,ID,XML,DTO,IO,FS,URI" />
+      <property name="allowedAbbreviations" value="AT,IT,DTO,A,I" />
```

Comment:
I 100% agree with adding `A` and `I` and `AT`. 
I also notionally agree with removing ID, IO, XML and URI however these are used frequently throughout a number of our code bases and it may be desirable to keep the style consistent, despite it being against the style guide. @spmason explicitly added these to the allowedAbbreviations list in this PR https://github.com/BrandwatchLtd/super-linter-action/pull/5/files https://github.com/BrandwatchLtd/super-linter-action/pull/5/files